modules/client/src/test/java/org/apache/ignite/client/fakes/FakeInternalTable.java

```diff
@@ -220,7 +220,7 @@ public CompletableFuture<Boolean> insert(BinaryRowEx row, @Nullable InternalTran
 
     /** {@inheritDoc} */
     @Override
-    public CompletableFuture<Collection<BinaryRow>> insertAll(Collection<BinaryRowEx> rows, @Nullable InternalTransaction tx) {
+    public CompletableFuture<List<BinaryRow>> insertAll(Collection<BinaryRowEx> rows, @Nullable InternalTransaction tx) {
```

Comment:
Your task is only about deleting, not about inserting data.

Reply:
There is the same problem. It's better to solve it here, isn't it?

Reply:
It's better to fix it on another ticket.

Reply:
I just chage a discription this one.

Reply:
I don’t see a problem if this is done in a separate task, so that it is clear why it was changed and what should happen in the end.

Reply:
I already added the necessity of fix all batch operation in the issue description.

Reply:
Then it’s worth correcting the ticket summary.